PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -77,27 +77,36 @@
         'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*'
     )
 )
-MicroEventContentMC = cms.PSet(
-    outputCommands = cms.untracked.vstring(MicroEventContent.outputCommands)
-)
-MicroEventContentMC.outputCommands += [
-        #'keep *_slimmedGenJets*_*_*',
-        'keep *_slimmedGenJets__*',
-        'keep *_slimmedGenJetsFlavourInfos_*_*',
-        'keep *_slimmedGenJetsAK8__*',
+
+MicroEventContentGEN = cms.PSet(
+    outputCommands = cms.untracked.vstring(
         'keep patPackedGenParticles_packedGenParticles_*_*',
         'keep recoGenParticles_prunedGenParticles_*_*',
         'keep LHEEventProduct_*_*_*',
-        'keep PileupSummaryInfos_slimmedAddPileupInfo_*_*',
         'keep GenFilterInfo_*_*_*',
         'keep GenLumiInfoHeader_generator_*_*',
         'keep GenLumiInfoProduct_*_*_*',
         'keep GenEventInfoProduct_generator_*_*',
+        'keep *_slimmedGenJetsFlavourInfos_*_*',
+        'keep *_slimmedGenJets_*_*',
+        'keep *_slimmedGenJetsAK8_*_*',
```

Comment:
please keep the ```__``` instead of ```_*_``` for slimmedGenJets and slimmedGenJetsAK8 as in the current baseline